dspace-api/src/main/java/org/dspace/content/authority/Concept.java

```diff
@@ -446,26 +446,38 @@ public static Concept find(Context context, int id) throws SQLException
             throws SQLException, AuthorizeException
     {
         ArrayList<Concept> concepts = new ArrayList<Concept>();
-	try {
-	    MetadataSchema mds = MetadataSchema.find(context, metadataSchema);
-	    MetadataField mdf = MetadataField.findByElement(context, mds.getSchemaID(), metadataElement, null);
-	    int target_field_id = mdf.getFieldID();
-	    log.info ("looking up concept metadata for " + searchString + " in field number " + target_field_id);
-	    TableRowIterator row = DatabaseManager.query(context, "select c.* from concept as c, conceptmetadatavalue as cmv where upper(cmv.text_value) = ? and cmv.parent_id = c.id and cmv.field_id = ?;", searchString, target_field_id);
-
-
-	    if (row == null) {
-		    return null;
-	    } else {
-		while(row.hasNext()) {
-		    concepts.add(new Concept(context,row.next()));
-		}
-	    }
-	} catch (NullPointerException e) {
-	    log.error("Unable to find concept by metadata: search=" + searchString + ", schema=" + metadataSchema + ", element=" + metadataElement, e);
-	}
-
-	return concepts;
+        try {
+            MetadataSchema mds = MetadataSchema.find(context, metadataSchema);
+            MetadataField mdf = MetadataField.findByElement(context, mds.getSchemaID(), metadataElement, null);
+            int target_field_id = mdf.getFieldID();
+            log.info ("looking up concept metadata for " + searchString + " in field number " + target_field_id);
+            TableRowIterator row = DatabaseManager.query(context, "select c.* from concept as c, conceptmetadatavalue as cmv where upper(cmv.text_value) = ? and cmv.parent_id = c.id and cmv.field_id = ?;", searchString, target_field_id);
+
+
+            if (row == null) {
+                return null;
+            } else {
+            while(row.hasNext()) {
+                concepts.add(new Concept(context,row.next()));
+            }
+            }
+        } catch (NullPointerException e) {
+            log.error("Unable to find concept by metadata: search=" + searchString + ", schema=" + metadataSchema + ", element=" + metadataElement, e);
+        }
+
+        return concepts;
+    }
+
+    public String getFullName(Context context) throws SQLException, AuthorizeException {
```

Comment:
@ryscher: I'm taking a new look at this comment and I don't think I understand why this would be journal-specific. Can you clarify?